packages/flutter/lib/src/material/scrollbar_theme.dart

```diff
@@ -293,3 +325,177 @@ class ScrollbarTheme extends InheritedWidget {
   @override
   bool updateShouldNotify(ScrollbarTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The leading and trailing Scrollbar buttons that are simple arrows.
+class ArrowScrollbarButtonsShape extends ScrollbarButtonsShape {
+  /// This abstract const constructor enables subclasses to provide
+  /// const constructors so that they can be used in const expressions.
+  const ArrowScrollbarButtonsShape();
+
+  @override
+  void paint(
+    Canvas canvas,
+    Rect leadingButtonRect,
+    bool isLeadingButtonHovered,
+    bool isLeadingButtonPressed,
+    Rect trailingButtonRect,
+    bool isTrailingButtonHovered,
+    bool isTrailingButtonPressed,
```

Comment:
Is it possible to just pass the current states of the leading button and the trailing button? That way you could just pass `_leadingButtonStates` and `_trailingButtonStates`

Reply:
Well gosh I actually recommended it. 🤦 
I think that doing this made it clearer that we should take a different approach here. Thanks for entertaining the suggestion, it made the code review easier. :)